### DISCUSSION TITLE
New Linux releases as AppImage

### DISCUSSION THREAD

**Discussion by Morphior** [f1multiviewer/issue-tracker#203] [Feature requests]
I propose providing new Linux releases as .AppImage files for ease of use. Currently you have to download the zip, create a new folder (I put it in ~/f1mv and call it the version number so for example `mkdir ~/f1mv/1.7.0`), unzip the files into the folder and then I have to tell my app launcher that there's a new executable and where exactly to find it. It's a hassle. Most program managers (I use rofi as app launcher and bauh as program manager) can just load an AppImage and then automatically tell the app launcher where to find it and which icon to use and so on.

**Comment by jeppeolesen** [comment]
Linux versions are now available in both .deb and .rpm, but the procedure remains the same for updating, namely manually downloading the new build.

We may look at distributing the Linux build via flatpak, but there are still some impediments to overcome before that happens.

See #562 

mod edit: typo

**Reply by marsmathis** [reply]
Thanks for the update! Since I’m using Arch (btw) and Multiviewer is in the AUR now, it’s all good.
